bindata/oauth-apiserver/deploy.yaml

```diff
@@ -58,12 +58,15 @@ spec:
               echo "Copying system trust bundle"
               cp -f /var/run/configmaps/trusted-ca-bundle/tls-ca-bundle.pem /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
             fi
+            echo -b "Rotating audit logs in /var/log/oauth-apiserver"
```

Comment:
I'm trying to get rid of the bash here (https://github.com/openshift/cluster-authentication-operator/pull/424), could this be done in an init container instead?